01/app.js

```diff
@@ -1 +1,34 @@
-console.log('DOM');
\ No newline at end of file
+console.log('DOM');
+
+
+const firstElement = document.querySelector(".comments__item ",".comments__item--newest")
+console.log(firstElement)
+
+const dataInfoEl = firstElement.querySelectorAll("span")
+
+console.log(dataInfoEl)
+
+const spanArray = Array.from(dataInfoEl);
+
+console.log(spanArray)
+
+spanArray.forEach(function(element,index){
+
+    if(spanArray && spanArray[index].hasAttribute("data-info")){
```

Comment:
Jeśli pętla działa to wiemy, że `spanArray` to tablica, więc nie ma potrzeby tego sprawdzać (dawać w warunku)